docs/dump_javadump.md

```diff
@@ -660,6 +678,206 @@ NULL           =================================
+4XENATIVESTACK               (0xB6680C0D [libj9dmp29.so+0x10c0d])
+4XENATIVESTACK               (0xB667F9D7 [libj9dmp29.so+0xf9d7])
+4XENATIVESTACK               (0xB6C52F6E [libj9prt29.so+0x1ef6e])
+4XENATIVESTACK               (0xB668B02F [libj9dmp29.so+0x1b02f])
+4XENATIVESTACK               (0xB668B4D3 [libj9dmp29.so+0x1b4d3])
+4XENATIVESTACK               (0xB66740F1 [libj9dmp29.so+0x40f1])
+4XENATIVESTACK               (0xB66726FA [libj9dmp29.so+0x26fa])
+4XENATIVESTACK               (0xB6C52F6E [libj9prt29.so+0x1ef6e])
+4XENATIVESTACK               (0xB66726A9 [libj9dmp29.so+0x26a9])
+4XENATIVESTACK               (0xB6676AE4 [libj9dmp29.so+0x6ae4])
+4XENATIVESTACK               (0xB668D75A [libj9dmp29.so+0x1d75a])
+4XENATIVESTACK               (0xB6A28DD4 [libj9vm29.so+0x81dd4])
+4XENATIVESTACK               (0xB6C52F6E [libj9prt29.so+0x1ef6e])
+4XENATIVESTACK               (0xB6A289EE [libj9vm29.so+0x819ee])
+4XENATIVESTACK               (0xB6A29A40 [libj9vm29.so+0x82a40])
+4XENATIVESTACK               (0xB6C52B6A [libj9prt29.so+0x1eb6a])
+4XENATIVESTACK               __kernel_rt_sigreturn+0x0 (0xB7747410)
+4XENATIVESTACK               (0xB75330B6 [libffi29.so+0x50b6])
+4XENATIVESTACK               ffi_raw_call+0xad (0xB7531C53 [libffi29.so+0x3c53])
+4XENATIVESTACK               (0xB69BE4AB [libj9vm29.so+0x174ab])
+4XENATIVESTACK               (0xB6A665BC [libj9vm29.so+0xbf5bc])
+4XENATIVESTACK               (0xB6A15552 [libj9vm29.so+0x6e552])
+4XENATIVESTACK               (0xB6A30894 [libj9vm29.so+0x89894])
+4XENATIVESTACK               (0xB6A6F169 [libj9vm29.so+0xc8169])
+4XENATIVESTACK               (0xB6C52F6E [libj9prt29.so+0x1ef6e])
+4XENATIVESTACK               (0xB6A6F1FA [libj9vm29.so+0xc81fa])
+4XENATIVESTACK               (0xB6A30994 [libj9vm29.so+0x89994])
+4XENATIVESTACK               (0xB6A2CE4C [libj9vm29.so+0x85e4c])
+4XENATIVESTACK               (0xB770487D [libjli.so+0x787d])
+4XENATIVESTACK               (0xB7719F72 [libpthread.so.0+0x6f72])
+4XENATIVESTACK               clone+0x5e (0xB763543E [libc.so.6+0xee43e])
+```
+
+The extract tells you that the current thread was `java/lang/Thread`, and information is provided about the Java callstack and Native callstack
+(`3XMTHREADINFO3`) at the point at which the crash occurred. To simulate a crash caused by a bug in customer or third party JNI code, our testcase
+calls into a JNI method whose native implementation causes a crash. The Java callstack shows the call into the JNI native method (`JNIcrasher`), and the
+native callstack shows the point of failure. In this example, the native call stack does not include any function names to help you isolate the error
+in the native code. The system dump that would have been produced alongside the Java dump can provide this information. Open the
+system dump with the [Dump viewer](tool_jdmpview.md) and use the `info thread` command to print the Java and native stack for the current thread.
```

Comment:
In general I'd expect the platform native debugger to do a better job on the native stack. Not sure if jdmpview does a reliable job as I've never tried this approach.